test/ios-project-service.ts

```diff
@@ -0,0 +1,148 @@
+/// <reference path=".d.ts" />
+"use strict";
+
+import Future = require("fibers/future");
+import * as path from "path";
+import temp = require("temp");
```

Comment:
Actually `temp.track` is used from another tests and the created directories are removed.